spanner/google/cloud/spanner_v1/gapic/spanner_client_config.py

```diff
@@ -16,6 +16,15 @@
                     "max_rpc_timeout_millis": 60000,
                     "total_timeout_millis": 600000
                 },
+                "streaming": {
+                    "initial_retry_delay_millis": 1000,
+                    "retry_delay_multiplier": 1.3,
+                    "max_retry_delay_millis": 32000,
+                    "initial_rpc_timeout_millis": 300000,
+                    "rpc_timeout_multiplier": 1.0,
+                    "max_rpc_timeout_millis": 300000,
+                    "total_timeout_millis": 3000000
+                },
```

Comment:
Is this manual hackage?  I don't see recent changes to the `googleapis` repository for spanner.

Reply:
Correct.

Reply:
Shouldn't this be done via `synth.py`, then?

Reply:
Making this change on our generated side. will strip from this pr once that is in.

Reply:
779569ac758dd0a789f4ed8c84c7656c28ca1de3